### ISSUE TITLE
Two flaky tests failing often on TC

### ISSUE THREAD

**Issue by warpech** [Starcounter/KitchenSink#181]
These two tests fail in 50% of time on TC:

```
AutoCompletePageTest(Firefox).AutoCompletePage_FillCountryNameThenSelectCountry
RedirectPageTest(Firefox).RedirectPage_ClickingOnFruitShouldChangeUrlAndText
```

Sometimes this fails:

```
CheckboxPageTest(Firefox).CheckboxPage_CheckboxUncheckedAndCheckedAgain
```

@un-tone do you have any idea why this might be? @PatrykSzwer was investigating this but could not find the reason: https://github.com/StarcounterApps/KitchenSink/pull/174#issuecomment-334487543

**Comment by un-tone** [comment]
There was 2 running `geckodriver.exe` processes in background. I guess it was a reason.

I also updated selenium binary and drivers to last versions on ScBuildAppWin01.

**Comment by warpech** [comment]
Thanks, but I think that was not it. A new build failed yesterday: https://teamcity.starcounter.org/viewLog.html?buildId=33879&buildTypeId=Apps_KitchenSink_Develop

Btw. Could you please update `CHANGELOG.txt` on the desktop of the build agent?

**Comment by warpech** [comment]
Fixed in #189

**Comment by un-tone** [comment]
> Could you please update CHANGELOG.txt on the desktop of the build agent?

Done.
